lib/packwerk.rb

```diff
@@ -14,8 +14,11 @@ module Packwerk
   extend ActiveSupport::Autoload
 
   # Public APIs
+  autoload :ApplicationLoadPaths
   autoload :Checker
   autoload :Cli
+  autoload :ConstantDiscovery
+  autoload :Node
```

Comment:
Needed to build a `Packwerk::Reference`, existing public API.

Reply:
Side-note, why is `Packwerk::Node` a class when it only nests `Location`? Can we make it a module?